src/asmcnc/apps/maintenance_app/widget_maintenance_spindle_settings.py

```diff
@@ -289,91 +295,97 @@
 
         Label:
             id: min_time_label
             x: cooldown_time_slider.pos[0]
             y: cooldown_time_slider.pos[1] - cooldown_time_slider.size[1] * 0.1
             size_hint: None, None
-            height: dp(30)
-            width: dp(30)
+            height: dp(0.0625*app.height)
+            width: dp(0.0375*app.width)
             color: hex('#888888ff')
-            font_size: dp(13)
+            font_size: dp(0.01625*app.width)
 
         Label:
             id: max_time_label
             x: cooldown_time_slider.pos[0] + cooldown_time_slider.size[0] * 0.9
             y: cooldown_time_slider.pos[1] - cooldown_time_slider.size[1] * 0.1
             size_hint: None, None
-            height: dp(30)
-            width: dp(30)
+            height: dp(0.0625*app.height)
+            width: dp(0.0375*app.width)
             color: hex('#888888ff')
-            font_size: dp(13)
+            font_size: dp(0.01625*app.width)
 
 
 
-""")
+"""
+)
 
-class SpindleSettingsWidget(Widget):
 
-    brand_list_sc1 = [' YETI SC1 digital 230V', ' YETI SC1 digital 110V', ' AMB digital 230V', ' AMB manual 230V', ' AMB manual 110V']
-    brand_list_sc2 = [' YETI SC2 digital 230V', ' YETI SC2 digital 110V'] + brand_list_sc1
+class SpindleSettingsWidget(Widget):
+    brand_list_sc1 = [
+        " YETI SC1 digital 230V",
+        " YETI SC1 digital 110V",
+        " AMB digital 230V",
+        " AMB manual 230V",
+        " AMB manual 110V",
+    ]
+    brand_list_sc2 = [
+                         " YETI SC2 digital 230V",
+                         " YETI SC2 digital 110V",
+                     ] + brand_list_sc1
 
     def __init__(self, **kwargs):
-    
         super(SpindleSettingsWidget, self).__init__(**kwargs)
-        self.sm=kwargs['screen_manager']
-        self.m=kwargs['machine']
-        self.l=kwargs['localization']
-
+        self.sm = kwargs["screen_manager"]
+        self.m = kwargs["machine"]
+        self.l = kwargs["localization"]
         self.rpm_override = self.m.spindle_cooldown_rpm_override
         self.cooldown_speed_slider.bind(value=self.cooldown_speed_updated)
         self.cooldown_time_slider.bind(value=self.cooldown_time_updated)
-
-        self.spindle_save_widget = widget_maintenance_spindle_save.SpindleSaveWidget(machine=self.m, screen_manager=self.sm, localization=self.l)
+        self.spindle_save_widget = widget_maintenance_spindle_save.SpindleSaveWidget(
+            machine=self.m, screen_manager=self.sm, localization=self.l
+        )
         self.spindle_save_container.add_widget(self.spindle_save_widget)
-
         self.update_strings()
 
+        self.wait_popup = InfoPopup(sm=self.sm, m=self.m, l=self.l,
+                                    main_string="SmartBench is raising the Z axis.",
+                                    title=self.l.get_str("Please wait") + "...",
+                                    popup_width=500, popup_height=200, button_one_text=None, auto_dismiss=True)
```

Comment:
Parameters here are quite different from existing please wait popup (i.e. different padding, delta). Has this been thoroughly proofed in all languages?